### PR TITLE
Bootstrap migrations

### PR DESCRIPTION
**Pull request by emord**
https://github.com/dimagi/commcare-hq/pull/8113#issuecomment-138335121

smsbillables should be the same

accounting is different though: https://gist.github.com/emord/357065f321ffdf72112a/revisions (left is the old south_migrations, right is new django migration)

the deleted code: https://gist.github.com/emord/357065f321ffdf72112a/revisions#diff-671811f11318cfeb2257a5ef5c20b500L26 looked like it was only migrating from old accounting models the new ones (it also references some fields that are no longer here)

@nickpell 

cc: @czue

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 5 errors:

* corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_mach_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_tropo_gateway.py, line 15 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py, line 18 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_unicel_gateway.py, line 15 - E501 line too long (124 > 115 characters)

File: corehq/apps/smsbillables/migrations/0002_bootstrap.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    Currency.objects.get_or_create(code='EUR')
+    Currency.objects.get_or_create(code='INR')
+
+
+def bootstrap_sms(apps, schema_editor):
+    bootstrap_grapevine_gateway(apps)
+    bootstrap_mach_gateway(apps)
+    bootstrap_tropo_gateway(apps)
+    bootstrap_twilio_gateway(apps)
+    bootstrap_unicel_gateway(apps)
```
Comment (nickpell) [inline corehq/apps/smsbillables/migrations/0002_bootstrap.py]: should also include the other management commands ```bootstrap_*``` and ```add_moz_zero_charge```

**Comment by czue** [comment]
(defer to @nickpell on this one)

**Comment by nickpell** [comment]
test failure looks real

**Comment by dimagimon** [comment]
There are 15 errors:

* corehq/apps/smsbillables/management/commands/add_moz_zero_charge.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway_update.py, line 16 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_mach_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_moz_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 17 - E221 multiple spaces before operator
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 17 - E501 line too long (145 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_test_gateway.py, line 17 - E501 line too long (144 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_test_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_tropo_gateway.py, line 15 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py, line 18 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway_incoming.py, line 16 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_unicel_gateway.py, line 15 - E501 line too long (124 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_yo_gateway.py, line 17 - E501 line too long (127 > 115 characters)

**Comment by dimagimon** [comment]
There are 15 errors:

* corehq/apps/smsbillables/management/commands/add_moz_zero_charge.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway_update.py, line 16 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_mach_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_moz_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 17 - E221 multiple spaces before operator
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 17 - E501 line too long (145 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_test_gateway.py, line 17 - E501 line too long (144 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_test_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_tropo_gateway.py, line 15 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py, line 18 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway_incoming.py, line 16 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_unicel_gateway.py, line 15 - E501 line too long (124 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_yo_gateway.py, line 17 - E501 line too long (127 > 115 characters)

**Comment by dimagimon** [comment]
There are 15 errors:

* corehq/apps/smsbillables/management/commands/add_moz_zero_charge.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_grapevine_gateway_update.py, line 16 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_mach_gateway.py, line 17 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_moz_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 17 - E221 multiple spaces before operator
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 17 - E501 line too long (145 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_telerivet_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_test_gateway.py, line 17 - E501 line too long (144 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_test_gateway.py, line 19 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_tropo_gateway.py, line 15 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway.py, line 18 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_twilio_gateway_incoming.py, line 16 - E501 line too long (127 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_unicel_gateway.py, line 15 - E501 line too long (124 > 115 characters)
* corehq/apps/smsbillables/management/commands/bootstrap_yo_gateway.py, line 17 - E501 line too long (127 > 115 characters)

**Comment by emord** [comment]
:white_check_mark:

**Comment by nickpell** [comment]
:+1:

**Pull request merged**